Tella/Components/BottomSheet/ConfirmBottomSheet.swift

```diff
@@ -24,32 +24,28 @@ struct ConfirmBottomSheet : View {
     }
     
     var contentView: some View {
-        
         VStack(alignment: .leading, spacing: 9) {
-            
+            Spacer(minLength: 10)
```

Comment:
I think it's unuseful to add a spacer on the top because we have already top padding 28 and it will affect another views